.travis.yml

```diff
@@ -18,3 +20,5 @@ services:
   - docker
 before_install:
   - (test $RUN_ON_LIVE_SERVER = 1 && ./.travis/start_hawkular_services.sh -d && ./.travis/wait_for_services.rb) || echo "Skipping live server"
+after_success:
+  - ./.travis.docs.rb
```

Comment:
Wondering if we could use the [deploy](https://docs.travis-ci.com/user/customizing-the-build/#Deploying-your-Code) phase of travis.

We would have less troubles if adding more jobs/permutations to the [build matrix](https://docs.travis-ci.com/user/customizing-the-build/#Build-Matrix).